app/assets/stylesheets/custom.scss

```diff
@@ -161,6 +161,28 @@ th {
   text-align: center;
 }
 
+// user order page
+.order-userinfo {
+  background-color: #e3e3e3;
+  border: 2px solid #6699ff;
+  border-radius: 20px;
+  margin-bottom: 2px;
+  min-height: 20px;
+  margin: auto;
+  padding: 6px;
+  width: 50%;
+}
+
+.order-total {
+  border: 2px solid #6699ff;
+  width: 50%;
```

Comment:
Properties should be ordered border, font-size, font-weight, margin, padding, text-align, width